### ISSUE TITLE
dynamic import is stage 4 (but still doesn't work)

### ISSUE THREAD

**Issue by dtudury** [standard/standard#1339]
**What version of standard?**
13.1.0

**What operating system, Node.js, and npm version?**
macOS 10.14.1, v10.16.0, 6.9.0

**What did you expect to happen?**
```
54:  const _installMap = new Map()
55:  async function _installComponent (path) {
56:    if (!_installMap.has(path)) {
57:      _installMap.set(path, import(path))
58:    }
59:    return _installMap.get(path)
60:  }
```
works on [83.29% of browsers](https://caniuse.com/#feat=es6-module-dynamic-import) and is in [stage 4 of the TC39 process](https://github.com/tc39/proposal-dynamic-import).
I expect standard to consider it valid code

**What actually happened?**
```
$ ./node_modules/.bin/standard
standard: Use JavaScript Standard Style (https://standardjs.com)
  /Users/dtudury/Repos/dtudury/rememberism/docs/controller.js:57:27: Parsing error: Unexpected token import
```

I know this has been reported before (#751) but that issue is closed and locked (because dynamic import was stage 3... but now it's stage 4)

**Comment by feross** [comment]
We're currently waiting on support from ESLint for dynamic import. See this issue: https://github.com/eslint/eslint/issues/11803 And here's a in-progress PR: https://github.com/eslint/eslint/pull/11983

Once that's been released, we'll update and finally get support :) In the meantime, you can use `babel-eslint` as a custom parser (instructions for how to set this up are in the README)

**Comment by dtudury** [comment]
sorry, yeah, I wouldn't have assumed it was your issue. however, searching the error message led to the closed issue where you kind of implied you'd fix it if it was in stage 4 😄

it sounds like the only purpose this issue serves is as a nag to merge the eslint version when it's released. (but maybe it feels good to close issues so I won't close it myself)

**Comment by feross** [comment]
Happy to leave this issue open as a reminder to watch the ESLint issue :)

**Comment by dtudury** [comment]
I just did a quick check, should this be working now? seems like everything got merged in

**Comment by feross** [comment]
@dtudury In order to get support, we need to update to ESLint 6.2.0 or later. There's a PR for that here: https://github.com/standard/standard/pull/1376 but it is blocked on this bug in `babel-eslint`:

- https://github.com/eslint/eslint/issues/12117
- https://github.com/babel/babel-eslint/issues/791

Once that's resolved, I'll update the version.

**Comment by feross** [comment]
Fixed in standard@14.1.0
